eng/source-build-tarball-build.yml

```diff
@@ -4,7 +4,7 @@ resources:
   pipelines:
   - pipeline: installer-build-resource
     source: installer
-    trigger: none
```

Comment:
This is not the correct change to enable tarball PR validation.

Here is the code that needs to be enabled/changed - https://github.com/dotnet/installer/blob/release/6.0.3xx/src/SourceBuild/Arcade/eng/common/templates/job/source-build-create-tarball.yml#L83

Additionally, we will likely need to disable offline until prebuilts are removed.  To do that, move this leg into the following if body - https://github.com/dotnet/installer/blob/release/6.0.3xx/src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml#L47